### PR TITLE
Fix input errors

### PR DESCRIPTION
**Pull request by frozzare**
**Type of change**

- [ ] New feature
- [x] Bug fix
- [ ] Security patch
- [ ] Documentation update

**Description**

Fix input error with 16 as prefix and check length. Tests are not working right now since it looks like [personnummer](https://github.com/personnummer/rust) has some issue with certain personnummer.

**Related issue**

**Motivation**

**Checklist**

- [x] I have read the **CONTRIBUTING** document.
- [x] I have read and accepted the **Code of conduct**
- [ ] Tests passes.
- [ ] Style lints passes.
- [x] Documentation of new public methods exists.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
 authors = ["Fredrik Forsmo <fredrik.forsmo@gmail.com>"]
+description = "Validate Swedish organization numbers"
 edition = "2021"
+homepage = "https://organisationsnummer.dev"
 license = "MIT"
+name = "organisationsnummer"
 readme = "README.md"
-homepage = "https://organisationsnummer.dev"
 repository = "https://github.com/organisationsnummer/rust"
+version = "1.0.0"
```
Comment (Johannestegner) [inline Cargo.toml]: Is this automatically updated? I expect so, but just want to doublecheck so we don't replace a package already in cargo :P

File: src/lib.rs [hunk 2]
```diff
@@ -67,9 +67,15 @@ impl TryFrom<&str> for Organisationsnummer {
             None => 0,
         };
 
+        let mut number = org.to_string().replace("-", "");
```
Comment (Johannestegner) [inline src/lib.rs:70]: I've replaced both + and - here, not sure if it's really needed, but just to make sure it works with a personnummer for a person whom is 100+ and is a EF, which I'm super sure happens a lot ;P

**Review by Johannestegner — APPROVED** [review]

**Comment by Johannestegner** [comment]
Tests seems to fail, so fix that, other than that, if my comments is irrelevant, LGTM ;)

File: Cargo.toml (hunk 1 above)
Comment (frozzare) [reply Cargo.toml]: Ah, no. I'll fix it!

File: src/lib.rs (hunk 2 above)
Comment (frozzare) [reply src/lib.rs:70]: It's fine, i'll try the number with personnummer package before trying to parse the organization number and those numbers can't have `+`

**Pull request merged**